src/views/Leads/index.js

```diff
@@ -198,7 +218,7 @@ export default class Leads extends React.PureComponent {
                 sortable: true,
                 order: 8,
                 modifier: row => (
-                    <div className={styles.confidentiality}>
+                    <div className="confidentiality">
```

Comment:
Why is a non-scoped className used here?

Reply:
The styles for Tables are stored in separate file (inside Tables/styles.scss) so non-scoped className.